content/influxdb/v2.0/tools/chronograf.md

```diff
@@ -53,15 +37,19 @@ For more information, see [InfluxQL support](/influxdb/v2.0/query-data/influxql/
       ```
 
     - **Connection Name:** Name to uniquely identify this connection configuration
-    - **Username:** InfluxDB username
-    - **Password:** InfluxDB [authentication token](/influxdb/v2.0/security/tokens/)
-    - **Telegraf Database Name:** Default database name
-    - **Default Retention Policy:** Default retention policy
+    - **Organization:** InfluxDB [organization](/influxdb/v2.0/organizations/)
+    - **Token:** InfluxDB [authentication token](/influxdb/v2.0/security/tokens/)
+    - **Telegraf Database Name:** database Chronograf uses to populate parts of
```

Comment:
Probably should be "bucket name" if using v2 auth?.... and no RP.  I'll need to add this as a separate issue for Chronograf.  For now, change the text to indicate, the Telegraf bucket name and recommendation is to leave the RP portion blank.